src/Schema/Events/Tap.ts

```diff
@@ -327,6 +327,50 @@ export interface TappedFairGroup extends TappedEntityGroup {
   action: ActionType.tappedFairGroup
 }
 
+/**
+ * A user taps a grouped explore module (City Guide, for example)
+ *
+ * This schema describes events sent to Segment from [[tappedEntityGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedCardsGroup",
+ *    context_module: "exploreBySection",
+ *    context_screen_owner_type: "home",
+ *    destination_screen_owner_type: "collectionsCategory",
+ *    destination_screen_owner_slug: "alex-katz-black-dress",
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface TappedCardsGroup extends TappedEntityGroup {
+  action: ActionType.tappedCardsGroup
+}
+
+/**
+ * A user taps a grouped explore module (City Guide, for example)
+ *
+ * This schema describes events sent to Segment from [[tappedEntityGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedCardsChipsGroup",
+ *    context_module: "discoverSomethingNewSection",
+ *    context_screen_owner_type: "home",
+ *    destination_screen_owner_type: "collection",
+ *    destination_screen_owner_id: "5359794d1a1e86c3740001f7",
+ *    destination_screen_owner_slug: "alex-katz-black-dress",
+ *    horizontal_slide_position: 1,
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface TappedCardsChipsGroup extends TappedEntityGroup {
```

Comment:
Still not sold on needing both these events. Maybe just start with `tappedCardGroup`? We'll be able to use the contextModule value to differentiate between specific section instances.

Reply:
makes sense, removed in the last commits 👍